src/software/new_geom/geom_constants.h

```diff
@@ -1,3 +1,5 @@
+#pragma once
```

Comment:
:scream:

Reply:
Seems like this is not causing issues for my PR anymore (maybe after changes to BUILD), but isn't it safer to put `pragma once` anyways?

Reply:
Yup yup, this is absolutely correct. No header guard can result in nasty errors.

See here for more info: https://www.learncpp.com/cpp-tutorial/header-guards/